widget/embedded/src/types/config.ts

```diff
@@ -8,22 +8,21 @@ import type { Asset } from 'rango-sdk';
  * @property {string} background
  * @property {string} foreground
  * @property {string} primary
- * @property {string} success
- * @property {string} error
- * @property {string} warning
- * @property {string} surface
- * @property {string} neutral
+ * @property {string} secondary
  *
  */
+export type Colors =
+  | 'background'
+  | 'foreground'
+  | 'primary'
+  | 'secondary'
+  | 'neutral';
```

Comment:
Could we use `export type Colors = keyof WidgetColors;` instead?

Reply:
fixed